### DISCUSSION TITLE
Hide tweets from blue subscribers

### DISCUSSION THREAD

**Discussion by Nifflas** [insin/control-panel-for-twitter#273] [Ideas]
I'm really frustrated with blue subscribers, mainly because they appear in top of all reply chains, and they are on average just way more annoying people than most twitter users. Would you consider the ability to hide all tweets from blue subscribers?

**Comment by Cpt-KeksMonster** [comment]
I thought the "Experiments" section thing would do that, but it doesn't do anything, at the moment. Hiding Blue subscribers would be a great option.

![grafik](https://user-images.githubusercontent.com/38458892/235167405-280a8515-eb4d-4ce3-bd2e-cae12267e585.png)
